composer.json

```diff
@@ -75,7 +75,8 @@
         "sonata-project/block-bundle": "<3.1.1 || >=4.0",
         "sonata-project/classification-bundle": "<3.0 || >= 5.0",
         "sonata-project/formatter-bundle": "<3.2",
-        "sonata-project/seo-bundle": "<2.1 || >=3.0"
+        "sonata-project/seo-bundle": "<2.1 || >=3.0",
+        "symfony/monolog-bundle": "<2.8 || >=3.0 <3.2"
```

Comment:
There is an operator missing

Reply:
I didn't test it but the [documentation](https://getcomposer.org/doc/04-schema.md#conflict) says this is an "and": 

> Note that when specifying ranges like `<1.0 >=1.1` in a `conflict` link, this will state a conflict with all versions that are less than 1.0 and equal or newer than 1.1 at the same time

Reply:
This is valid? WTF?